example/90-shoot.yaml

```diff
@@ -147,6 +147,15 @@ spec:
   # workersSettings:
   #   sshAccess:
   #     enabled: false
+# etcd:
+#   main:
+#     autoscaling:
```

Comment:
camelCase?

Reply:
I previously had this but then noticed that we have `coreDNS.autoscaling` ([ref](https://github.com/gardener/gardener/blob/48ee6e2bff6afa72b50e6f7b2d25db3abd938398/pkg/apis/core/v1beta1/types_shoot.go#L1805)), so we should probably be consistent?!